homeassistant/components/sonos/speaker.py

```diff
@@ -591,6 +582,15 @@ async def async_rebooted(self) -> None:
     #
     # Battery management
     #
+    @soco_error()
+    def fetch_battery_info(self) -> dict[str, Any]:
+        """Fetch battery_info for the speaker."""
+        battery_info = self.soco.get_battery_info()
```

Comment:
This used to trap `contextlib.suppress(ConnectionError, TimeoutError, SoCoException)`. Is that not an issue anymore ?